### DISCUSSION TITLE
new release looks good

### DISCUSSION THREAD

**Discussion by qpwo** [nicoespeon/abracadabra#614] [General]
https://github.com/nicoespeon/abracadabra/releases/tag/6.14.0

I'll always be pleased about improvements to 'move to existing file'

**Comment by nicoespeon** [comment]
Fair! 

I would also like to improve the "Move Statements" refactorings. It's not fluid enough and I find myself not using it as much as I would… because it doesn't _feel_ safe enough.

Inline Function is another one I want to get another shot at.

But for the moment, I'm working on something a bit different. Hopefully, I'll tell you more about it soon. It will be part of Abracadabra. It's not a refactoring, but it should help with refactoring 😄

**Reply by nicoespeon** [reply]
I forgot to ever come back to this discussion, but what I was referring to was ["Highlight Identifiers"](https://github.com/nicoespeon/abracadabra?tab=readme-ov-file#highlight-identifiers) 😄
